### DISCUSSION TITLE
Question : How to learn botorch

### DISCUSSION THREAD

**Discussion by YouHyin** [meta-pytorch/botorch#1651] [Q&A]
Hello, I'm a beginner at botorch. I want to perform Bayesian optimization very skillfully using Botorch, but I feel that it is very difficult with tutorial alone. So I wonder if there are any lectures or books related to this. Also, what method do you recommend to fully learn botorch?
Should I follow the tutorial? Ask for advice on what to do.

Thank you!

**Comment by eytan** [comment]
Do you want to apply Bayesian optimization to specific problems you have or
do you want to do research on Bayesian optimization (eg develop new models
and acquisition functions and maybe publish papers about them?)

Best
E

**Reply by YouHyin** [reply]
I want to apply Bayesian optimization to my own objective functions. Also, I plan to publish a papaer while doing this research through botorch!

**Reply by Balandat** [reply]
If you just want to run Bayesian Optimization in a user-friendly fashion you should take a look at Ax (https://ax.dev/) - it's powered by BoTorch under the hood. 

If you do want to learn more about Bayesian Optimization itself, a couple of good starting points are a classic book on Gaussian Processes (https://gaussianprocess.org/gpml/) and a more recent tutorial (with lots of references): https://arxiv.org/abs/1807.02811
